src/components/screens/IndexScreen/Integrations.js

```diff
@@ -0,0 +1,186 @@
+import Cypress from '../../../images/integrations/cypress.svg';
+import Emotion from '../../../images/integrations/emotion.svg';
+import Figma from '../../../images/integrations/figma.svg';
+import Invision from '../../../images/integrations/invision.svg';
+import Jasmine from '../../../images/integrations/jasmine.svg';
+import Jest from '../../../images/integrations/jest.svg';
+import Nextjs from '../../../images/integrations/nextjs.svg';
+import Notion from '../../../images/integrations/notion.svg';
+import Playwright from '../../../images/integrations/playwright.svg';
+import Puppeteer from '../../../images/integrations/puppeteer.svg';
+import Sass from '../../../images/integrations/sass.svg';
+import Sketch from '../../../images/integrations/sketch.svg';
+import Tailwind from '../../../images/integrations/tailwind.svg';
+import TestingLib from '../../../images/integrations/testing-lib.svg';
+import Vite from '../../../images/integrations/vite.svg';
+import Zeplin from '../../../images/integrations/zeplin.svg';
+import Zeroheight from '../../../images/integrations/zeroheight.svg';
+
+const { color, spacing, pageMargin, pageMargins, breakpoint } = styles;
+
+const Title = styled(Subheading)`
+  display: block;
+  margin-bottom: 1.5rem;
+  color: ${color.dark};
+  @media (min-width: ${breakpoint * 1}px) {
+    margin-bottom: 2rem;
+  }
+`;
+
+const IntegrationItem = styled.div`
+  width: 48px;
+  height: 48px;
+
+  @media (min-width: ${breakpoint * 1.333}px) {
+    width: 60px;
+    height: 60px;
+  }
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 10px;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.05), 0 5px 15px 0 rgba(0, 0, 0, 0.1);
+
+  img {
+    width: 100%;
+    height: 100%;
+    display: block;
+  }
+
+  ${(props) =>
+    props.inverse &&
+    css`
+      svg path,
+      svg circle {
+        fill: ${color.lightest};
+      }
+    `}
+`;
+
+const IntegrationList = styled.div`
+  margin: 0 auto 16px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px 30px;
+  justify-content: center;
+  max-width: 516px;
+  align-items: start;
+  @media (min-width: ${breakpoint * 1.333}px) {
+    margin: 0 auto 30px;
+    gap: 30px 40px;
+    max-width: 660px;
+  }
+
+  &:last-child {
+    margin-bottom: 0 !important;
+  }
+`;
+
+const IntegrationsWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  flex: 1;
+`;
+
+const Inner = styled.div`
+  align-items: center;
+  text-align: center;
+
+  padding: 3rem ${spacing.padding.medium}px;
+  @media (min-width: ${breakpoint * 1}px) {
+    /* margin: 0 ${pageMargin * 3}%; */
+    padding-bottom: 5rem;
+  }
+  @media (min-width: ${breakpoint * 2}px) {
+    /* margin: 0 ${pageMargin * 4}%; */
+  }
+`;
+
+const Wrapper = styled.div``;
+
+export default function CTA({ ...props }) {
+  return (
+    <Wrapper {...props}>
+      <Inner>
+        <Title>Integrations</Title>
+        <IntegrationsWrapper>
+          <IntegrationList>
```

Comment:
Would love to turn all these into links (to addons or elsewhere). If you're worried about them getting stale then we can turn the whole section into a link to `/addons`

Reply:
Yea that's fair, can you make the change?